src/ServiceFabricPubSub/SubscriberService/PackageRoot/Config/Settings.xml

```diff
@@ -1,16 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <Settings xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://schemas.microsoft.com/2011/01/fabric">
-  <!-- This is used by the StateManager's replicator. -->
-  <Section Name="ReplicatorConfig">
-    <Parameter Name="ReplicatorEndpoint" Value="ReplicatorEndpoint" />
```

Comment:
It's part way towards a .NET core refactor, its work in progress - will port over.